### PR TITLE
Docs for Tulipa

### PR DESCRIPTION
**Pull request by joaquimg**
@datejada, do you have some suggestions for the "structure" section?

Other suggestions are appreciated.

More on #25

### CODE REVIEW

File: TulipaEnergyModel/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+The GitHub organization is [https://github.com/TulipaEnergy](https://github.com/TulipaEnergy)
+
+## License
+
+TulipaEnergyModel is licensed under the [Apache License 2.0](https://github.com/TulipaEnergy/TulipaEnergyModel.jl/blob/main/LICENSE).
+
+## Structure
+
+The example at `main.jl` was inspired in code extracted from ...
```
Comment (datejada) [inline TulipaEnergyModel/README.md]:
```suggestion
The example cases in this directory are taken from [https://github.com/TulipaEnergy/TulipaEnergyModel.jl/tree/main/benchmark/EU](https://github.com/TulipaEnergy/TulipaEnergyModel.jl/tree/main/benchmark/EU) 

The EU case is a stylized European-level investment and operation modified to consider integer investments and unit commitment variables. In addition, the code at `main.jl` modifies the timesteps to consider different problem sizes ranging from a day to a year.
```

Comment (datejada) [reply TulipaEnergyModel/README.md]: Thanks, @joaquimg. I added information for the structure section.

**Comment by odow** [comment]
@datejada did you say you left some comments here? If so, perhaps you didn't submit the review?

**Review by datejada — CHANGES_REQUESTED** [review]

**Comment by datejada** [comment]
> @datejada did you say you left some comments here? If so, perhaps you didn't submit the review?

Yeap, I forgot to submit it. Sorry about that. It is done, now. Thanks!

**Pull request merged**